src/components/project.tsx

```diff
@@ -1,35 +1,27 @@
 /** @jsx jsx */
-import { jsx, Heading } from "theme-ui"
-import { MDXRenderer } from "gatsby-plugin-mdx"
-import Layout from "./layout"
-import SEO from "./seo"
-import { graphql } from "gatsby"
+import React from "react";
```

Comment:
This is just renaming the old project template getting confused here so you can ignore this.